specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,3027 @@
+              }
+            ]
+          }
+        }
+      },
+      "required": [
+        "analysisInput",
+        "tasks"
+      ]
+    },
+    "TaskIdentifier": {
+      "type": "object",
+      "description": "Base task object.",
+      "properties": {
+        "taskId": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "taskId"
+      ]
+    },
+    "TaskParameters": {
+      "type": "object",
+      "description": "Base parameters object for a text analysis task.",
+      "properties": {
+        "loggingOptOut": {
+          "type": "boolean",
+          "default": false
+        }
+      }
+    },
+    "PreBuiltTaskParameters": {
+      "type": "object",
+      "description": "Parameters object for a text analysis task using pre-built models.",
+      "properties": {
+        "modelVersion": {
+          "type": "string",
+          "default": "latest"
+        }
+      }
+    },
+    "PreBuiltResult": {
+      "properties": {
+        "errors": {
+          "type": "array",
+          "description": "Errors by document id.",
+          "items": {
+            "$ref": "#/definitions/DocumentError"
+          }
+        },
+        "statistics": {
+          "$ref": "#/definitions/RequestStatistics"
+        },
+        "modelVersion": {
+          "type": "string",
+          "description": "This field indicates which model is used for scoring."
+        }
+      },
+      "required": [
+        "errors",
+        "modelVersion"
+      ]
+    },
+    "CustomTaskParameters": {
+      "type": "object",
+      "description": "Parameters object for a text analysis task using custom models.",
+      "properties": {
+        "projectName": {
+          "type": "string"
+        },
+        "deploymentName": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "projectName",
+        "deploymentName"
+      ]
+    },
+    "CustomResult": {
+      "properties": {
+        "errors": {
+          "type": "array",
+          "description": "Errors by document id.",
+          "items": {
+            "$ref": "#/definitions/DocumentError"
+          }
+        },
+        "statistics": {
+          "$ref": "#/definitions/RequestStatistics"
+        },
+        "projectName": {
+          "type": "string",
+          "description": "This field indicates the project name for the model."
+        },
+        "deploymentName": {
+          "type": "string",
+          "description": "This field indicates the deployment name for the model."
+        }
+      },
+      "required": [
+        "errors",
+        "projectName",
+        "deploymentName"
+      ]
+    },
+    "CustomEntitiesTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Custom Entities task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomTaskParameters"
+        },
+        {
+          "$ref": "#/definitions/EntitiesTaskParameters"
```

Comment:
CustomEntitiesTaskParameters is getting a `modelVersion` parameter from EntitiesTaskParameters. I think it might need to be removed since this is a custom task.

Reply:
I've just exchanged `EntitiesTaskParameters` with an explicit setting of `stringIndexType` to follow convention that I've been using for other tasks which use this parameter.